frontend/public/components/volumes-table.tsx

```diff
@@ -56,6 +56,7 @@ const VolumeRow: React.FC<VolumeRowProps> = ({value, pod}) => {
   const kind = _.get(getVolumeType(value.volume), 'id', '');
   const loc = getVolumeLocation(value.volume);
   const name = value.name;
+  const namespace = pod.metadata.namespace;
```

Comment:
You want to make sure you're getting the namespace from the original resource, not the pod template. I don't think it's always set here.